sdk/mx.sdk/mx_sdk_vm_impl.py

```diff
@@ -1248,6 +1249,8 @@ def contents(self):
             elif isinstance(image_config, mx_sdk.LauncherConfig):
                 suffix = _exe_suffix
                 project_name_f = GraalVmLauncher.launcher_project_name
+                if mx.get_os() == 'linux' and mx.get_os_variant() == 'musl':
+                    build_args.append('-H:+StaticExecutableWithDynamicLibC')
```

Comment:
Hmm, do we hit issues with the executables we produce without this option? If I understand right, this means that all native-image executables that we ship as part of Graal only depend on `musl` and statically link everything else

Reply:
Ah, perfect - thank you! :)